doc/gov/chapters/chapter04.md

```diff
@@ -4,66 +4,114 @@
 ## 4.1 Introduction
 
+The CNTT is structured into Workstreams so it can scale and accommodate participation from the wider community. The CNTT is open to participation by any individual or organization that agrees to comply with its Term of Reference ([ToR](../../../GSMA_CNTT_Terms_of_Reference.pdf)) (each such participant, a “Participant”). Each of the Workstreams within CNTT has one Lead and up to two Co-Leads. Leads and Co-Leads for each Workstream are selected by the community itself through nomination and voting processes. This document explains the CNTT policies and process regarding nomination and selection.
+
 <a name="4.2"></a>
 ## 4.2 Workstreams Lead/Co-Lead Nomination and Selection
 
+For each Workstream, following leadership positions exist:
+- Lead.
+- Up to two Co-leads. 
+
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
+The length of term for all CNTT leadership roles is **12** calender months.
+
 <a name="4.2.2"></a>
 ### 4.2.2 Representation
 
+Any Workstream Leads or Co-Leads positions can be held by any Participant.
+
+<a name="4.2.3"></a>
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or another participant for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
+- An individual can self-nominate themselves or another participant to multiple positions.
+
+<a name="4.2.4"></a>
+### 4.2.4 Selection
+
+There will be a nomination window during which individuals can nominate others or nominate themselves for a position. The start and end date of the nomination window will be determined by Technical or Governance Committee Leads(depending on the nature of the Workstream).
+
+During the nomination period, Participants can send their nominations to LFN Staff who will help administering the process. The details and email addresses of LFN staff will be clarified when the process is started.
+
+Once the nomination period expires for a particular Workstream, the selection process will start as follows:
+
+- Selection process for Leads positions and Co-Leads will occur separately.
+- No voting will be required if number of nominations received for a particular position is less than or equal to the number of available vacancies. 
+- Online Voting process will take place to select candidate(s) from the nominee list for any given position.
+  - The exact tool used for voting will be determined by the LFN staff who will help administering the process and will depends on the approved tool at that point of time.
+- Only Voting representatives can vote (from the up to date voting representative list).
+- Candidate(s) who receives the highest number of votes win.
+
+<a name="4.2.5"></a>
+### 4.2.5 Interim Leads/Co-Leads
+
+- In case a Lead steps down, an interim person (preferably from the existing Co-leads) will be assigned by Technical Steering/Governance Committee to take over position duties until next available election.
```

Comment:
Suggest delete "available" from this and the next two sentences

Reply:
Fixed in https://github.com/cntt-n/CNTT/pull/896/commits/ec67ca2c13373ce0007d4dd83d74d0b4b54e9360